docs/cli_reference.md

```diff
@@ -96,16 +96,16 @@ Will list information on all active service deployments available in `YOUR_NAMES
 ### Delete Services
 
 ```shell
-$ bodywork service display \
-    --namespace=YOUR_NAMESPACE
+$ bodywork service delete \
+    --namespace=YOUR_NAMESPACE \
     --name=SERVICE_NAME
 ```
 
 Delete an active service deployment - e.g. one that is no longer required for a project.
 
 ## Manage Cronjobs
 
-Bodywork can schedule workflows to run periodically to a pre-defined schedule, using remote workflow-controllers.
+Bodywork can schedule workflows to run periodically on a schedule. These workflows are managed by workflow-controllers started on your k8s cluster.
```

Comment:
Maybe make it clearer that these workflow controllers are automatically deployed/started by bodywork.

e.g. These workflows are managed by the workflow-controllers automatically started on your k8s cluster by bodywork.

Reply:
Agreed.